external/storm-kafka/src/jvm/storm/kafka/FailedMsgRetryManagerConfig.java

```diff
@@ -0,0 +1,7 @@
+package storm.kafka;
+
+import java.io.Serializable;
+
+public interface FailedMsgRetryManagerConfig extends Serializable {
```

Comment:
Since this a factory pattern, I would rename this `FailedMsgRetryManagerFactory`, and the impl `ExponentialBackoffMsgRetryManagerFactory`.